File: src/kt/ifelse.kt
fun main(args: Array<String>) {
    var value = true;
    if (value) {
        println("Got true")
    } else {
        println("Got else")
    }

    value = false
    if (value)
        println("Got true")
    else
        println("Got else")

    var result = 3
    if (result == 1)
        print("Got true")
    else if (result == 2)
        print("Got else")
    else
        print("Got if else")
}

File: src/c/ifelse.c
#include <stdio.h>

main()
{
    int value = 1;
    if (value)
    {
        printf("Got true\n");
    }
    else
    {
        printf("Got else\n");
    }

    value = 0;
    if (value)
        printf("Got true\n");
    else
        printf("Got else\n");

    int result = 3;
    if (value == 1)
        printf("Got true");
    else if (result == 2)
        printf("Got else");
    else
        printf("Got if else");
}